project/scripts/cmdTests

```diff
@@ -56,6 +56,7 @@ cp tests/neg/i6371/B_2.scala $OUT/B.scala
 "$SBT" "scalac $OUT/A.scala -d $OUT1"
 rm $OUT/A.scala
 "$SBT" "scalac -classpath $OUT1 -d $OUT1 $OUT/B.scala" > "$tmp" 2>&1 || echo "ok"
+cat "$tmp" # for debugging
```

Comment:
Could you comment that out or only display it in case the test doesn't pass? Otherwise we get some error output in the CI that could be misinterpreted as a test failure.

Reply:
Comented out